docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -161,6 +161,7 @@ property key | Description | Default |
 `plugin.kafka.topic_profilings` | Specify which Kafka topic name for Thread Profiling snapshot to report to. | `skywalking_profilings` |
 `plugin.kafka.topic_management` | Specify which Kafka topic name for the register or heartbeat data of Service Instance to report to. | `skywalking_managements` |
 `plugin.springannotation.classname_match_regex` |  Match spring beans with regular expression for the class name. Multiple expressions could be separated by a comma. This only works when `Spring annotation plugin` has been activated. | `All the spring beans tagged with @Bean,@Service,@Dao, or @Repository.` |
+`plugin.toolkit.log.transmit_preformatted` | Whether or not to transmit logged data as pre-formatted or template + argument(s). | `true` |
```

Comment:
I think we need to be clear about `pre-formatted or template + argument(s)`. Add the description in the toolkit doc or here?

Reply:
I've added additional notes to the log4j 2.x and logback 1.x toolkit docs.